pkgs/by-name/hy/hyp2mat/package.nix

```diff
@@ -25,20 +26,27 @@ stdenv.mkDerivation rec {
     sha256 = "03ibk51swxfl7pfrhcrfiffdi4mnf8kla0g1xj1lsrvrjwapfx03";
   };
 
+  postPatch = lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    substituteInPlace doc/Makefile.am --replace-fail \
+      '$(HELP2MAN) --output=$@ --no-info --include=$< $(PROGNAME)' \
+      '$(HELP2MAN) --output=$@ --no-info --include=$< ${buildPackages.hyp2mat}/bin/`basename $(PROGNAME)`'
+  '';
+
   nativeBuildInputs = [
     autoreconfHook
+    bison
+    flex
+    gengetopt
+    groff
+    help2man
     pkg-config
   ];
 
   buildInputs = [
     libharu
     libpng
     zlib
-    bison
     flex
```

Comment:
Do we need flex here, too?

Reply:
Apparently not. Sometimes it is required to find the header.